README.md

```diff
@@ -10,22 +10,24 @@ Currently we only support a few entities and per entity only a few actions:
 3.  Deal (create, update, move)
 4.  Deal phases (get)
 5.  Deal sources (get)
-6.  Activity types (get)
-7.  Event (create, update, delete, get, getById)
-8.  Business types (get)
-9.  Tags (get)
-10. Invoice (get)
-11. Credit notes (get)
-12. Payment terms (get)
-13. Tax rates (get)
-14. Withholding tax rates (get)
-15. Departments (get)
-16. User (me, get, getById)
-17. Webhook (get, register)
-18. Projects (get, getById)
-19. Tasks (get, getById)
-20. Milestones (get, getById)
-21. TimeTracking (get, getById)
+6.  Deal (get, getById)
```

Comment:
Can you merge this with number 3?

Reply:
Done (https://github.com/cschalenborgh/teamleader-php-sdk/blob/feature-support-more-entities/README.md)